app/scripts/components/dashboard/charts-container.jsx

```diff
@@ -7,16 +7,30 @@ require('stylesheets/dashboard/charts-container');
 const ChartsContainer = React.createClass({
   propTypes: {
     children: PropTypes.node.isRequired,
+    onToggle: PropTypes.func.isRequired,
+    state: PropTypes.object.isRequired,
   },
   render() {
+    const [ stateClass, below ] = this.props.state.match({
+      Open: () => [ 'open', this.props.children ],
+      Closed: () => [ 'closed', []],
+    });
     return (
-      <div className="charts-container">
-        <div className="charts-container-header">
-          <span className="tab-label">
-          <T k="charts.toggle.activate" /></span>
-          <ViewMode />
+      <div className={`charts-container ${stateClass}`}>
+        <div className="above">
+          <div className="charts-container-header">
+            <span
+                className="tab-label"
+                onClick={this.props.onToggle}
+                role="button">
+              <T k="charts.toggle.activate" />
+            </span>
+            <ViewMode />
```

Comment:
ViewMode should be ouside of the charts-containter-header, shouldn't it?
Let's leave it and change it on the future